Library/Formula/desk.rb

```diff
@@ -0,0 +1,18 @@
+class Desk < Formula
+  desc "Lightweight workspace manager for the shell"
+  homepage "https://github.com/jamesob/desk"
+  url "https://github.com/jamesob/desk/archive/v0.3.1.tar.gz"
+  sha256 "b687e2cfa742f763d689391f67a5b5225324e282a0fed100487b1570988d7758"
+
+  def install
+    bin.install "desk"
+  end
+
+  test do
+    system "mkdir", "-p", testpath/".desk/desks"
```

Comment:
Ah, thanks @bfontaine. I'm not that up-to-date on many of ruby's system utils. `Pathname` looks awesome, though!